duckduckgo/Android
# updated system search activity to match the current designs

Task/Issue URL: https://app.asana.com/0/715106103902962/1199987030166438

**Description**:
Our interstitial search screen looks inconsistent with other UI in our app.
Updating this UI will make using the interstitial search feel more similar to what users are already used to in our app.

**Steps to test this PR**:
1. Add the DDG widget to the home screen
2. Tap on it to launch a search
3. Check new designs

<img width="1400" alt="image" src="https://user-images.githubusercontent.com/531613/114191660-f2807080-994c-11eb-9787-8c6891c023ed.png">

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
